src/Mesh/generic.jl

```diff
@@ -227,3 +227,43 @@ function getNodalLaplacianMatrix(Mesh::AbstractTensorMesh)
 	end
 	return Mesh.nLap	
 end
+
+
+"""
+function jInv.Mesh.getBoundaryNodes(Mesh)
+	
+	Returns an array tuple (boundary node indices, inner node indices)
+	
+	Input: 
+		Mesh::Abstract Mesh
+	
+	Output:
+		Tuple{Array{Int64,1},Array{Int64,1}}
+"""
+function getBoundaryNodes(M::AbstractMesh)
```

Comment:
should we specify this for tensor meshes? If so, shall we move this to tensor.jl?